pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -111,14 +217,58 @@ func GetTemplateData(config *baremetal.Platform, networks []types.MachineNetwork
 	templateData.ExternalStaticDNS = config.BootstrapExternalStaticDNS
 	templateData.ExternalMACAddress = config.ExternalMACAddress
 
-	_, externalURLv6 := externalURLs(config.APIVIPs)
+	// If the user has manually set disableVirtualMediaTLS to False in the Provisioning CR, then enable TLS in ironic.
+	// The default value of 'true' is temporary (for backporting to older releases), which will be changed to 'false' in the future.
+	templateData.DisableIronicVirtualMediaTLS = true
+	protocol := "http"
+
+	provisioningCR := struct {
+		spec struct {
+			disableVirtualMediaTLS *bool `yaml:"disableVirtualMediaTLS"`
+		} `yaml:"spec"`
+	}{}
+	openshiftManifests := &manifests.Openshift{}
+	logrus.Debugf("Getting Openshift manifests dependency")
+	dependencies.Get(openshiftManifests)
+	logrus.Debugf("After getting Openshift manifests dependency")
+	var provisioningCRBytes []byte
+	for _, file := range openshiftManifests.Files() {
+		logrus.Debugf("openshiftManifests.Files() - file: %s", file.Filename)
+		if file.Filename == "openshift/99_baremetal-provisioning-config.yaml" {
```

Comment:
Will fix after testing end-to-end

Reply:
I wish this were its own asset so you didn't have to do this, but at the moment there appears to be no choice :slightly_frowning_face:

I'm not convinced that we shouldn't add this field to the install-config so that we can access the data directly.